.gitlab-ci.yml

```diff
@@ -35,7 +35,10 @@ variables:
 # at the same location whoever triggers a pipeline.
   CUSTOM_CI_BUILDS_DIR: /usr/workspace/umdev/gitlab-runner
 # Tells Gitlab to recursively update the submodules when cloning the project.
```

Comment:
Update the comment, too. I guess we don't need submodules of submodules because we are using spack to build the dependencies?

Reply:
Exactly, incidentally, it saves time and traffic through the firewall.

Reply:
Thanks!